content/4_Gold/Divisibility.mdx

```diff
@@ -3,7 +3,7 @@ id: divisibility
 redirects:
   - /gold/divis
 title: 'Divisibility'
-author: Darren Yao, Michael Cao, Andi Qu, and Kevin Sheng
+author: Darren Yao, Michael Cao, Andi Qu and Kevin Sheng
```

Comment:
Why'd you change this? Isn't the comma correct (it's used on every other page?)

Reply:
Ah sorry. I removed it

Reply:
Oops, I meant you should use comma, like so 😅 
`Darren Yao, Michael Cao, Andi Qu, and Kevin Sheng`